misc/scripts/copyright_headers.py

```diff
@@ -35,58 +36,61 @@
 /**************************************************************************/
 """
 
-fname = sys.argv[1]
+if len(sys.argv) <= 2:
```

Comment:
Shouldn't this check here bef or < 2? currently it exits if its processing only a single header